plugins/GSdx/GSHwHack.cpp

```diff
@@ -58,11 +58,17 @@ bool GSC_DBZBT2(const GSFrameInfo& fi, int& skip)
 	{
 		if(fi.TME && /*fi.FBP == 0x00000 && fi.FPSM == PSM_PSMCT16 &&*/ (fi.TBP0 == 0x01c00 || fi.TBP0 == 0x02000) && fi.TPSM == PSM_PSMZ16)
 		{
-			if (Dx_only) // Feel like texture shuffle but not sure
-				skip = 26; //27
+			// Alpha channel (red line issue has been fixed).
+			// Sky texture (Depth) is properly rendered on OpenGL only for the NTSC version. The PAL version still has some issues (half screen bottom issue).
+			if (g_crc_region == CRC::EU || Dx_only)
+			{
+				skip = 26; // 27
```

Comment:
Maybe remove the `// 27`?

Reply:
Test both, see what's better, and keep only that value then. Comment was introduced in 357030330a587aa51e1461a0b8009919b40be594.

Reply:
Based on the commit message the 27 value was used when it only worked on the PAL version. Either it was just a general correction, or using the higher value removed something it shouldn't on the NTSC versions.

Reply:
> or using the higher value removed something it shouldn't on the NTSC versions

Yeah it was that. Feast your eyes on this, looks lovely doesn't it with 27 :smile: 

![pcsx2 2018-02-05 15-30-10-92](https://user-images.githubusercontent.com/18107717/35809913-5cf7f80c-0a8a-11e8-8035-d75212556c29.png)